pkg/controller/utils/utils.go

```diff
@@ -19,10 +19,21 @@ const (
 	Finalizer = "finalizer.aws.managed.openshift.io"
 	WaitTime  = 25
 
-	// EnvDevMode is the name of the env var we set to run locally and to skip
-	// initialization procedures that will error out and exit the operator.
-	// ex: `FORCE_DEV_MODE=local operatorsdk up local`
-	EnvDevMode = "FORCE_DEV_MODE"
+	// envDevMode is the name of the env var we set to indicate we're running in a development
+	// environment vs. production. Set it to one of the DevMode* consts defined below.
+	// For example:
+	// * Running locally: `FORCE_DEV_MODE=local operator-sdk up local`
+	// * Running in a cluster:
+	//   . Edit deploy/operator.yaml. Under .spec.template.spec.env, add:
+	//         - name: FORCE_DEV_MODE
+	//           value: cluster
+	//   . `oc apply` all the YAML files in deploy/, including the updated operator.yaml.
+	envDevMode = "FORCE_DEV_MODE"
+
+	// MetricsPort is the port at which we'll serve metrics
+	MetricsPort = "8080"
```

Comment:
Are these ever going to be used anywhere but cmd/manager/main.go?

Reply:
No; I only moved them here so they would be near the [comment](https://github.com/openshift/aws-account-operator/pull/407/files/9922a4fb3b2e578e868e7469ab32ce921e79014f#diff-c9c1114922f55187dc190facb2951ff9R56-R59) that talks about them. I can move it back if you don't like that (and reword the comment to refer to main.go, which I thought was slightly ickier at the time, but meh).

Reply:
Since they're specific to main, I think it makes sense to keep them there, and keep utils cleaner as well. But I won't block the PR on it

Reply:
Sure, no big deal, I moved 'em back. Makes the diff smaller too :)